src/rules/no-flowmax-in-forwardref.coffee

```diff
@@ -0,0 +1,23 @@
+module.exports =
+  meta:
+    docs:
+      description: 'Flag uses of flowMax() within forwardRef()'
+      category: 'Possible Errors'
+      recommended: yes
+    schema: [
+      type: 'object'
+      additionalProperties: no
+    ]
+    fixable: 'code'
+
+  create: (context) ->
+    report = (node) ->
+      context.report {
+        node,
+        message: "flowMax() cannot be used within forwardRef()"
```

Comment:
I'd probably "soften" the error message a bit - technically you *can* use `flowMax()` within `forwardRef()`, it's just almost always a bad idea

How about `Avoid using flowMax() within React.forwardRef()`?